src/components/AudioPlayer/Player/Player.component.js

```diff
@@ -4,31 +4,37 @@ import propTypes from 'prop-types';
 import {
   RepeatButton,
   RewindButton,
-  PlayContainer,
+  PlayButton,
   ForwardButton,
   ShuffleButton,
 } from '../PlayerButtons';
 
 const PlayerComponent = ({
   isDisabled,
-  audioRef,
-  toggleIsRandom,
+  toggleRandom,
+  toggleRepeat,
+  fastForward,
+  rewind,
+  play,
 }) => (
   <div className={`buttons ${isDisabled ? 'disable-buttons' : ''}`}>
     <div className="buttons-container">
-      <RepeatButton />
-      <RewindButton />
-      <PlayContainer audioRef={audioRef} />
-      <ForwardButton />
-      <ShuffleButton onClick={toggleIsRandom} />
+      <RepeatButton onClick={toggleRepeat} />
+      <RewindButton onClick={rewind} />
+      <PlayButton onClick={play} />
+      <ForwardButton onClick={fastForward} />
+      <ShuffleButton onClick={toggleRandom} />
```

Comment:
`onRepeatClick`, `onRewindClick`, `onPlayClick`, `onForwardClick`, `onShuffleClick`.

Reply:
This may be a good place to apply currying functions, thoughts? @tonymtz , `const onClick = action => () => { ... }`

Reply:
Go for it!

Reply:
On it!